com.rustdesk.RustDesk.json

```diff
@@ -0,0 +1,116 @@
+      "buildsystem": "simple",
+      "build-commands": [
+        "make -j4 && PREFIX=./build make install",
+        "cp -r ./build/*  /app/"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/jordansissel/xdotool/releases/download/v3.20211022.1/xdotool-3.20211022.1.tar.gz",
+          "sha256": "96f0facfde6d78eacad35b91b0f46fecd0b35e474c03e00e30da3fdd345f9ada"
+        }
+      ]
+    },
+    {
+      "name": "pam",
+      "buildsystem": "simple",
+      "build-commands": [
+        "./configure --disable-selinux --prefix=/app && make -j4 install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/linux-pam/linux-pam/releases/download/v1.3.1/Linux-PAM-1.3.1.tar.xz",
+          "sha256": "eff47a4ecd833fbf18de9686632a70ee8d0794b79aecb217ebd0ce11db4cd0db"
+        }
+      ]
+    },
+    {
+      "name": "libarchive",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--disable-static"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/libarchive/libarchive/releases/download/v3.5.1/libarchive-3.5.1.tar.gz",
+          "sha256": "9015d109ec00bb9ae1a384b172bf2fc1dff41e2c66e5a9eeddf933af9db37f5a"
+        }
+      ]
+    },
+    {
+      "name": "rustdesk",
+      "buildsystem": "simple",
+      "build-commands": [
+        "bsdtar -zxvf rustdesk.deb",
+        "tar -xvf ./data.tar.xz",
+        "rm -rf usr/share/icons/hicolor/scalable/",
+        "cp -r usr/*  /app/",
+        "mkdir -p /app/bin && ln -s /app/lib/rustdesk/rustdesk /app/bin/rustdesk"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "only-arches": [
+            "aarch64"
+          ],
+          "url": "https://github.com/rustdesk/rustdesk/releases/download/1.2.4/rustdesk-1.2.4-aarch64.deb",
+          "sha256": "4f8be7199a1e8acf0799ce8cea6af08bcf2d6f073a3a098809ac842e3a87d1e8",
+          "dest-filename": "rustdesk.deb",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": "309593",
+            "url-template": "https://github.com/rustdesk/rustdesk/releases/download/$version/rustdesk-$version-aarch64.deb"
+          }
+        },
+        {
+          "type": "file",
+          "only-arches": [
+            "x86_64"
+          ],
+          "url": "https://github.com/rustdesk/rustdesk/releases/download/1.2.4/rustdesk-1.2.4-x86_64.deb",
+          "sha256": "43c2f43ad7a8a173fa51093944243f04f003f675e74812644780eee1468b3f9b",
+          "dest-filename": "rustdesk.deb",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": "309593",
+            "url-template": "https://github.com/rustdesk/rustdesk/releases/download/$version/rustdesk-$version-x86_64.deb"
+          }
+        }
+      ]
+    }
+  ],
+  "rename-desktop-file": "rustdesk.desktop",
+  "rename-icon": "rustdesk",
+  "cleanup": [
```

Comment:
This should be at the top after finish-args.

Reply:
Please modify it directly.

Reply:
https://github.com/flathub/flathub/pull/5233/commits/fbc0d9561df79b61a80095ea6a2239508a68abe8